### DISCUSSION TITLE
How can be manage the expenses in the version 6.7?

### DISCUSSION THREAD

**Discussion by ciucaadrian11** [Intermesh/groupoffice#991] [Q&A] [answered]
Hello,
Please help me with my nonunderstanding. How I can introduce the expenses in the new version 6.7? In the new Finance module I don't find eny function (module) in order to introduce the expenses compare with the old version 6.6 where at the Billing module, there is a section for recording expenses.
Thank you!

**Comment by mschering** [comment, marked as answer]
Hi,

You can create purchase invoices and orders to register expenses.

Best regards,
Merijn

**Reply by ciucaadrian11** [reply]
Thank you Merijn Schering!
